### PR TITLE
removed PDFreactor from codebase, including lib in env PYTHONPATH

### PR DESCRIPTION
**Pull request by kave**
@rosskarchner @kurtw 

Included PDFreactor python lib to python path on Flapjack
`export PYTHONPATH="${PYTHONPATH}:/opt/PDFreactor/wrappers/python/lib"`

### CODE REVIEW

**Comment by jimmynotjim** [comment]
Is it gone forever or just while we figure out it's issues? (crossing my fingers it's forever)

**Comment by kave** [comment]
@jimmynotjim no we're still using it lol, but realized that we did not need to include it in the codebase when we can reference it from the pdfreactor installation location on the server

**Comment by jimmynotjim** [comment]
Shucks, guess we should still keep that pdfreactor css task. Looks good to me :+1:

**Comment by rosskarchner** [comment]
You're on the right track, but I think this trades one assumption (the exact version of PDFReactor) with another (the exact location of PDFReactor on the filesystem).

Here's an approach I've been thinking about:

- we create a settings.base variable (like PDFREACTOR_LIB) , that points to the directory. (perhaps itself, configurable with an environment variable, and defaulting to '/opt/PDFreactor/wrappers/python/lib' ).
- When core.services is imported, it checks if PDFREACTOR_LIB is set and if so, it appends it to sys.path. 
- If `import PDFReactor` fails, we should recover, and all further attempts to generate PDF's should generate a sensible exception (like PDFReactorNotConfigured)

**Comment by kave** [comment]
@rosskarchner yea that sounds like a good approach, i'll implement it

File: cfgov/core/services.py [hunk 1]
```diff
@@ -10,10 +12,17 @@
...
+
+class PDFReactorNotConfigured(Exception):
+    pass
+
 ## TODO: Update to python 3 when PDFreactor's python wrapper supports it.
 if six.PY2:
-    from core.lib.PDFreactor import *
-
+    try:
+        sys.path.append(os.environ.get('PDFREACTOR_LIB'))
```
Comment (rosskarchner) [inline cfgov/core/services.py]: Does that work with the default https://github.com/cfpb/cfgov-refresh/pull/1075/files#diff-ac27c5919c46548eada7a7388e726c59R156 

I think sys.path entries need to be directories. (so, the 'lib' directory containing PDFReactor.py)

Comment (kave) [inline cfgov/core/services.py]: yea i was under that impression too, but tested it out locally and it works.

pull it out of the pdfreactor docs
```
import sys
sys.path.append("path/to/PDFreactor.py/")
from PDFreactor import *
```

Comment (rosskarchner) [inline cfgov/core/services.py]: Cool!

**Comment by rosskarchner** [comment]
:+1:

**Pull request merged**